src/command_modules/azure-cli-ams/azure/cli/command_modules/ams/_params.py

```diff
@@ -78,6 +78,14 @@ def load_arguments(self, _):  # pylint: disable=too-many-locals, too-many-statem
         c.argument('output_index', help='The element index of the output to remove.',
                    type=int, default=None)
 
+    with self.argument_context('ams transform output add') as c:
+        c.argument('preset', help='Allowed values: {}. In addition to the allowed values, you can also pass the local full path to a custom preset JSON file.'
+                   .format(", ".join(get_presets_definition_name_completion_list())))
+        c.argument('audio_insights_only', arg_group='Video Analyzer', action='store_true', help='Use this flag to only extract audio insights when processing a video file.')
+        c.argument('audio_language', arg_group='Audio/Video Analyzer', help='The language for the audio payload in the input using the BCP-47 format of \"language tag-region\" (e.g: en-US). The list of supported languages are: en-US, en-GB, es-ES, es-MX, fr-FR, it-IT, ja-JP, pt-BR, zh-CN')
```

Comment:
Why not using a list and joining?

Reply:
agree, you can place the list in _sdk_utils.py and import it from there